### PR TITLE
Natalsa566 rainfall

### PR DESCRIPTION
**Pull request by Natalsa566**

### CODE REVIEW

File: src/main/java/org/academy/kata/implementation/Natalsa566/SixImpl.java [hunk 1]
```diff
@@ -14,13 +17,50 @@ public String balance(String book) {
     public double f(double x) {
         return 0;
     }
-
+@Override
```
Comment (lhalam) [inline src/main/java/org/academy/kata/implementation/Natalsa566/SixImpl.java:20]: remove this line

File: src/main/java/org/academy/kata/implementation/Natalsa566/SixImpl.java [hunk 2]
```diff
@@ -14,13 +17,50 @@ public String balance(String book) {
...
+                    .map(record -> Double.parseDouble(record.split(" ")[1]))
+                    .mapToDouble(Double::doubleValue)
+                    .toArray();
+        } catch (NumberFormatException e) {
+            return Double.NaN;
+        }
+        return Arrays.stream(rainfallValues).average().orElse(Double.NaN);
     }
-
+    @Override
```
Comment (lhalam) [inline src/main/java/org/academy/kata/implementation/Natalsa566/SixImpl.java:37]: remove this line

File: src/main/java/org/academy/kata/implementation/Natalsa566/SixImpl.java (hunk 1 above)
Comment (Natalsa566) [reply src/main/java/org/academy/kata/implementation/Natalsa566/SixImpl.java]: removed

Comment (Natalsa566) [reply src/main/java/org/academy/kata/implementation/Natalsa566/SixImpl.java]: removed

**Pull request merged**